apps/portal/examples/postgrest-template/content/source/Dockerfile.migrate

```
@@ -7,7 +7,7 @@ FROM golang:1.22-alpine AS builder
 RUN apk add --no-cache wget
 
 # Download golang-migrate binary
-RUN wget -qO- https://github.com/golang-migrate/migrate/releases/download/v4.17.0/migrate-linux-amd64.tar.gz | tar xz -C /tmp/
+RUN wget -qO- https://github.com/golang-migrate/migrate/releases/download/v4.17.0/migrate.linux-amd64.tar.gz | tar xz -C /tmp/
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)